src/core/xfa/template.js

```diff
@@ -2266,7 +2266,7 @@ class Image extends StringObject {
         c.charCodeAt(0)
       );
       const blob = new Blob([buffer], { type: this.contentType });
-      html.attributes.src = URL.createObjectURL(blob);
+      html.attributes.src = URL.createObjectURL(blob).href;
```

Comment:
It isn't correct.
You can try with this form: https://www.canada.ca/content/dam/ircc/migration/ircc/english/pdf/kits/forms/imm1344e.pdf, there is a picture in top-left corner with a canadian flag.
When dumping `html.attributes.src`, I got an `undefined`.
Anyway the picture is correctly displayed so it likely means that the URL is correctly cloned (or I missed something).
The failing URL in the document you mentioned is a local path `Q:\hrmgmt\New HR NEU Logos\NortheasternUniv-w-Seal-Red.jpg`.

Reply:
Sorry, that was a bit too much copy-and-paste which I've fixed now; thank you!

Obviously `URL.createObjectURL` returns a string, but the rest of the patch is necessary to prevent breaking errors.